roles/aws_setup_credentials/README.md

```diff
@@ -1,4 +1,5 @@
-# aws_setup_credentials
+aws_setup_credentials
+=====================
```

Comment:
These should be equivalent.

Reply:
Using two different types of heading styles was giving a Markdown linting error locally, we should pick one style and use it consistently within a document. So I updated this one to match the style used in the rest of the document.